setup.py

```diff
@@ -51,11 +51,11 @@
         "addfips~=0.3.1",
         "catalystcoop.dbfread~=3.0",
         "coloredlogs~=15.0",
-        "dask>=2021.8,<2023.0",
+        "dask>=2021.8,<2022.5.1",
         "datapackage~=1.11",  # Transition datastore to use frictionless.
         # "email-validator>=1.0.3",  # pydantic[email] dependency
-        "fsspec>=2021.7,<2023.0",  # For caching datastore on GCS
-        "gcsfs>=2021.7,<2023.0",  # For caching datastore on GCS
+        "fsspec>=2021.7,<2022.3.1",  # For caching datastore on GCS
+        "gcsfs>=2021.7,<2022.3.1",  # For caching datastore on GCS
```

Comment:
why are these changing as well?

Reply:
I wasn't clear on how the calendar versioning works, and it's different from semantic versioning. Breaking changes can come with any new version. Having a tighter range of versions here means that we'll get a PR from the dependabot any time a new version is released, so we'll at least know that something has changed, and get to see the CI results before we accept the new version.